### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by rnshah9**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+project: rnshah9/jpeg-compressor
+target: jpge
```
Comment (xansec) [inline Mayhemfile:2]: Looks like a typo?

**Review by xansec — CHANGES_REQUESTED** [review]
Could you please resolve

Multi-stage build
Correct usage of filepath
Generic project name

File: Mayhemfile (hunk 1 above)
Comment (rnshah9) [reply Mayhemfile:2]: I might be missing something obvious. Which typo?

Comment (xansec) [reply Mayhemfile:2]: jpge -> jpeg. I'll resolve.

Comment (xansec) [reply Mayhemfile:2]: Ah wait, that's actually what the target is called.

**Comment by xansec** [comment]
LGTM!

**Pull request merged**